node-it/src/test/scala/com/wavesplatform/it/api/AsyncHttpApi.scala

```diff
@@ -1027,5 +1028,25 @@ object AsyncHttpApi extends Assertions {
       transactions.broadcast(SignedTransaction.of(Some(unsigned), Seq(ByteString.copyFrom(proofs))))
     }
 
+    def broadcastInvokeScript(caller: KeyPair,
```

Comment:
Move the whole NodeExtGrpc to another file, please

Reply:
done